govc/namespace/vmclass/change.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright (c) 2024-2024 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vmclass
+
+import (
+	"context"
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/namespace"
+)
+
+type change struct {
+	*flags.ClientFlag
+
+	spec namespace.VirtualMachineClassUpdateSpec
+}
+
+func init() {
+	cli.Register("namespace.vmclass.change", &change{})
+}
+
+func (cmd *change) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
+	cmd.ClientFlag.Register(ctx, f)
+
+	f.Int64Var(&cmd.spec.CpuCount, "cpus", 0, "The number of CPUs.")
+	f.Int64Var(&cmd.spec.MemoryMb, "memory", 0, "The amount of memory (in MB).")
+}
+
+func (cmd *change) Process(ctx context.Context) error {
+	return cmd.ClientFlag.Process(ctx)
+}
+
```

Comment:
You can add `Usage` func, which indicates in the help an argument is required:
```go
func (*change) Usage(context.Context) string {
	return "NAME"
}
```
Then in the help text you'll see:
```diff
-Usage: govc namespace.vmclass.change [OPTIONS]
+Usage: govc namespace.vmclass.change [OPTIONS] NAME
```

Reply:
Thanks Doug, fixed!